_posts/2018-12-06.Rust-1.31.md

```diff
@@ -0,0 +1,338 @@
+---
+layout: post
+title: "Announcing Rust 1.31"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.31.0. Rust is a
```

Comment:
I think we should have 2018 edition in the first sentence - anything else feels like burying the lede